src/Resources/views/CRUD/tree.html.twig

```diff
@@ -58,7 +58,7 @@ file that was distributed with this source code.
                         <ul class="dropdown-menu" role="menu">
                             {% for context in contexts %}
                                 <li>
-                                    <a href="{{ admin.generateUrl('tree', { 'context': context.id }) }}">
+                                    <a href="{{ admin.generateUrl(action|default('tree'), { 'context': context.id }) }}">
```

Comment:
There is `This template is not used at all` in the comments. Can we be sure `tree` is the value passed to the template ?

Also, I think we should deprecate this template because we don't use it.